pycompwa/pycompwa/plotting/__init__.py

```diff
@@ -1,6 +1,5 @@
 import matplotlib.pyplot as plt
-# from matplotlib import rc
-from matplotlib.colors import LinearSegmentedColormap
+
```

Comment:
I would like to avoid modules containing a single file

Reply:
I forgot to add the `rootplotdatareader.py` file, I will update the pull request shortly